<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TriggerSetupDialog</class>
 <widget class="QDialog" name="TriggerSetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="analog">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Analog trigger</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="margin">
           <number>4</number>
          </property>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Value: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="valueSlider">
              <property name="maximum">
               <number>255</number>
              </property>
              <property name="value">
               <number>50</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="valueSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
              <property name="value">
               <number>50</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Slope:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="risingRadio">
              <property name="text">
               <string>Rising</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="fallingRadio">
              <property name="text">
               <string>Falling</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup</string>
              </attribute>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Type:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="edgeRadio">
              <property name="text">
               <string>Edge</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup_2</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="thresholdRadio">
              <property name="text">
               <string>Threshold</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup_2</string>
              </attribute>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="digital">
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="margin">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Digital trigger</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Format:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="formatCombo"/>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Number of samples:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="samplesCombo"/>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Sample 1:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="sample1Edit"/>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Sample 2:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="sample2Edit"/>
            </item>
           </layout>
          </item>
          <item row="4" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Sample 3:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="sample3Edit"/>
            </item>
           </layout>
          </item>
          <item row="5" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Sample 4:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="sample4Edit"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TriggerSetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TriggerSetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>valueSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>valueSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>valueSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>valueSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
